content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst

```diff
@@ -1,116 +1,144 @@
+<inventory/shipping_receiving/configure-weight>`.
 
-In the **Inventory** module, click on :menuselection:`Configuration --> Settings`.
-Under **Shipping Connectors**, flag the transportation companies you want to
-integrate :
+For detailed instructions and context on configuring these options, refer to :doc:`this document
+<../setup_configuration/third_party_shipper>`.
 
-.. image:: labels/labels03.png
+Install shipping connector
+--------------------------
+
+Install the shipping connector's module in Odoo to integrate Odoo with the third-party shipping
+carrier. To do so, navigate to :menuselection:`Inventory app --> Configuration --> Settings`.
+
+Under the :guilabel:`Shipping Connectors` heading, select the third-party shipping carrier's
+checkbox to install it. Multiple shipping connectors may be selected. Then, click :guilabel:`Save`.
+
+.. image:: ../setup_configuration/third_party_shipper/shipping-connectors.png
    :align: center
+   :alt: Options of available shipping connectors in Odoo.
 
-Then click on **Apply**.
+Delivery method
+---------------
 
-Configure the delivery method
------------------------------
+Next, add the carrier details by going to :menuselection:`Inventory app --> Configuration -->
+Shipping Methods`.
 
-To configure your delivery methods, go to the **Inventory** module, click on
-:menuselection:`Configuration --> Delivery Methods`.
+On the :guilabel:`Shipping Methods` page, choose the desired delivery method. For details on
+configuring the third party shipping method, go to:
 
-The delivery methods for the chosen shippers have been automatically
-created.
+.. seealso::
+   - :doc:`Configure third-party carrier <../setup_configuration/third_party_shipper>`
+   - :doc:`DHL credentials <../setup_configuration/dhl_credentials>`
+   - :doc:`Sendcloud credentials <../setup_configuration/sendcloud_shipping>`
+   - :doc:`UPS credentials <../setup_configuration/ups_credentials>`
+
+Then, select the :guilabel:`Get Rate and Create Shipment` option in the :guilabel:`Integration
+Level` field to ensure the shipping label gets printed.
 
-.. image:: labels/labels01.png
+.. image:: labels/integration-level.png
    :align: center
+   :alt: Set the "Get Rate and Create Shipment" option.
+
+Production environment
+~~~~~~~~~~~~~~~~~~~~~~
 
-In the **Pricing** tab, the name of the provider means that the delivery
-will be handled and computed by the shipper system.
+With the delivery method details configured, click the :guilabel:`Test Environment` smart button to
+set it to production environment.
 
-The configuration of the shipper is split into two columns :
+.. warning::
+   Setting the delivery method to :guilabel:`Production` creates **real** shipping labels, and users
+   are at risk for being charged through their carrier account (e.g., UPS, FedEx), **before** users
+   charge customers for shipping. Verify all configurations are correct before launching the
+   delivery method to :guilabel:`Production`.
```

Comment:
```suggestion
   delivery method to :guilabel:`Production Environment`.
```
I am suggesting the addition of 'Environment' -- and adding capitalization -- because that's how it appears on the UI/screen